src/highlight/HighlightList.tsx

```diff
@@ -23,15 +65,14 @@ export function HighlightList({ annotations, className, onSelect }: Props): JSX.
     useOutsideEvent('mouseup', rootElRef, (): void => setIsListening(true));
 
     return (
-        <svg
-            ref={rootElRef}
-            className={classNames(className, 'ba-HighlightList', { 'is-listening': isListening })}
-            data-resin-component="highlightList"
-        >
-            {annotations.map(({ id, target }) => (
-                <HighlightAnnotation key={id} annotationId={id} onSelect={onSelect} rects={target.shapes} />
-            ))}
-        </svg>
+        <div className={classNames('ba-HighlightList', className)} data-resin-component="highlightList">
+            <HighlightCanvas shapes={canvasShapes} />
+            <HighlightSvg ref={rootElRef} className={classNames({ 'is-listening': isListening })}>
+                {sortedAnnotations.map(({ id, target }) => (
+                    <HighlightTarget key={id} annotationId={id} onSelect={onSelect} rects={target.shapes} />
```

Comment:
Should this be `shapes` rather than `rects` to stay consistent with HighlightCanvas?